config/m4/ucs.m4

```diff
@@ -150,6 +150,36 @@ CHECK_CROSS_COMP([AC_LANG_SOURCE([static int rc = 1;
                              supports __attribute__((constructor))])]
                 )
 
+
+#
+# Manual configuration of cacheline size
+#
+
+AC_ARG_WITH([cache-line-size],
+        [AC_HELP_STRING([--with-cache-line-size=SIZE],
+            [Build UCX with cache line size defined by user. This parameter
+             overwrites defaulf cache line sizes defines in
+             UCX (x86-64: 64, Power: 128, ARMv8: 64/128). The supported values are: 64, 128])],
+        [],
+        [with_cache_line_size=no])
+
+AS_IF([test "x$with_cache_line_size" != xno],[
+	     case ${with_cache_line_size} in
+                 64)
```

Comment:
Do you ever notice warnings in configure output ?

On Fri, Sep 2, 2016 at 4:50 AM, Mike Dubman <notifications@github.com>
wrote:

> In config/m4/ucs.m4
> <https://github.com/openucx/ucx/pull/939#discussion_r77321347>:
>
> > +
> > +#
> > +# Manual configuration of cacheline size
> > +#
> > +
> > +AC_ARG_WITH([cache-line-size],
> > +        [AC_HELP_STRING([--with-cache-line-size=SIZE],
> > +            [Build UCX with cache line size defined by user. This parameter
> > +             overwrites defaulf cache line sizes defines in
> > +             UCX (x86-64: 64, Power: 128, ARMv8: 64/128). The supported values are: 64, 128])],
> > +        [],
> > +        [with_cache_line_size=no])
> > +
> > +AS_IF([test "x$with_cache_line_size" != xno],[
> > +	     case ${with_cache_line_size} in
> > +                 64)
>
> and warn if unexpected value differ from traditionally known sizes.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openucx/ucx/pull/939/files/014e6a9871bdfe1b3fbd48b1e2df7a406fa60d2c#r77321347>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACIe2MDv_ZpPEXlCMAZwb1pnRLvU4dg_ks5ql_FJgaJpZM4JyAwV>
> .
>